drivers/hv/hv_common.c

```diff
@@ -340,8 +340,7 @@ u8 __init get_vtl(void)
 	if (hv_result_success(ret)) {
 		ret = output->as64.low & HV_VTL_MASK;
 	} else {
-		pr_err("Failed to get VTL(error: %lld) exiting...\n", ret);
```

Comment:
How are we planning to deal with a legitimate error scenario ?
Can we return 0 in case non Hyper-V hypervisors from this function ?